docs/developers/protocol/hermez-protocol/circuits/circuits.md

```diff
@@ -401,6 +395,37 @@ Besides, if coordinator does not fulfill all the possible recipient to receive f
 |:------------:|:-----:|:--------------:|
 | newStateRoot | field | new state root |
 
+### compute-fee
+#### Description
+Computes the final amount of fee to apply given the fee selector
+
+- Steps:
+  - selects fee factor, `feeOut`, to apply given `feeSel` and `applyFee`
+  - compute `feeOutNotShifted = amount * feeOut` and convert it into bits in `feeOutBits[253]`
+  - compute `applyShift` to decide if shift has to be applied to `feeOutNotShifted`
+  - select bits on `feeOutBits[253]` depending on `applyShift` flag
+  - assert `feeOut` is $< 2^{129}$
```

Comment:
```suggestion
  - assert `feeOut` is $< 2^{192}$
```

Reply:
Maximum fee amount to apply will be 2^129